### ISSUE TITLE
Uninterruptible Power Supply (UPS) for Protecting Networking Equipment

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#167]
While reading the setup instructions for the Ubiquiti Unifi POE Switch, I noted the recommendation for a UPS:
<img width="942" alt="ubiquiti-recommends-ups-for-switch" src="https://user-images.githubusercontent.com/194400/101813762-13f35c00-3b15-11eb-89b4-9714286789cd.png">

I had thought about getting a UPS before but so far I've just thought that our network equipment would run of our house batteries ... but the more I think about it, I think the network equipment should have it's own UPS because that way the security (cameras) and internet have their own dedicated power in the event of a cut. 💭 

Did some quick research and read a few forum threads.
https://community.ui.com/questions/UPS-for-Unifi-system/109b5273-09ea-4e8d-81d8-133e07065870
Basically we don't need anything special, just something that protects from current overloads and outages.

https://www.kuantokusta.pt/informatica/Perifericos/UPS-Fontes-de-Alimentacao-Ininterrupta/Eurotech-Smart-UPS-1000VA-p-1-380232
![image](https://user-images.githubusercontent.com/194400/101813306-7a2baf00-3b14-11eb-9164-c24e5131f8b4.png)

The _cheapest_ appears to be **El Corte Ingles**: https://www.elcorteingles.pt/feed/A34349835
![image](https://user-images.githubusercontent.com/194400/101815181-15258880-3b17-11eb-8253-e5c9f539cc77.png)

**Comment by iteles** [comment]
What's the question here? I say let's just order the UPS from ECInglés 👍

**Comment by nelsonic** [comment]
@iteles Yeah, I discussed it with Hugo (Electrician) and he agrees it's a good idea to have the UPS.
Please order from ECInglés. Thanks. 👍

**Comment by iteles** [comment]
Purchased.

Looks like it might take a while to arrive:
![Uploading image.png…]()

**Comment by nelsonic** [comment]
@iteles a delayed arrival/delivery is fine. 👍 
It's not _urgent_ it's a _very_ nice to have that will ensure our security systems have backup power. 🔋

**Comment by nelsonic** [comment]
The UPS Arrived but sat in a box because it wasn't given a priority... ⏳

![image](https://user-images.githubusercontent.com/194400/233787828-5753eeda-5a4d-4091-884c-3da27ca70c20.jpeg)

I've unboxed now as part of: https://github.com/dwyl/learn-computer-networking/issues/14
And am measuring its dimensions so that I can mount it on the board. 

It only has two power sockets so im going to connect it to the Unifi Switch 16 and the ISP router so that if power goes down these two devices continue to function for ~6h (depending on how many more things we connect to the PoE Switch...)

**Comment by nelsonic** [comment]
Didn't have to _manually_ cut the power to test this.
Our workmen outside have an extension connected the workshop and overloaded the circuit. 
The breaker was tripped and all the appliances in the basement were turned off.
The UPS kicked in and the ISP Modem/Router kept working 
while I restored the power by flipping the breaker switch in the consumer unit.
I call this a success. 🎉 

Closing. ✅

**Comment by iteles** [comment]
Yay but also 🙄
